site/cds_rdm/tasks.py

```diff
@@ -86,10 +86,15 @@ def sync_local_accounts_to_names(since=None, user_id=None):
         current_app.logger.debug(
             f"Names sync | Fetching active users updated since {since}."
         )
-        users = User.query.filter(
-            User.updated > since,
-            User.active == True,
-        ).all()
+        if since:
```

Comment:
Sure! I would simply update the logging as well to be more clear:
```python
if since:
        current_app.logger.debug(
            f"Names sync | Fetching active users updated since {since}."
        )
        ...
else:
        current_app.logger.debug(
            f"Names sync | Fetching all active users."
        )
        ...
```